pkg/couchdb/couchdb.go

```diff
@@ -839,6 +857,30 @@ func DefineIndexes(g *errgroup.Group, db Database, indexes []*mango.Index) {
 	}
 }
 
+// Copy copies an existing doc to a specified destination
+func Copy(db Database, doctype, path, destination string) (map[string]interface{}, error) {
+	headers := make(map[string]string)
```

Comment:
Maybe we can use the shorter `headers := map[string]string{"Destination": destination}`